mockery/parse.go

```diff
@@ -25,6 +25,20 @@ func NewParser() *Parser {
 }
 
 func (p *Parser) Parse(path string) error {
+	abs, err := filepath.Abs(path)
```

Comment:
Rather than introduce another variable for people to think about, it'd be simpler to just update `path` here.